server.js

```diff
@@ -20,7 +20,8 @@ app.post("/login", validateLogin(), user.loginUser);
 
 app.post("/users/:id/responders", middleware.checkToken, user.addResponders);
 
-app.post("/users/:id/location", middleware.checkToken, user.addLocation);
+app.post("/users/:id/location", middleware.checkToken, user.updateLocation);
```

Comment:
If we are updating location, shouldnt it be a PUT/PATCH? I realize i have let that slip for the previous POST endpoint to be pushed to master. Also have you thought of how we are going to get the user's location when they signup?

Reply:
Yeah! I could definitely make it a PUT instead, I think I was just blindly following the previous patterns when I was doing this :zipper_mouth_face:

Regarding the user's location on signup, I've left it blank for now - I'm not too sure but I dont know about getting the user's location in the background? In the front end, the app automatically gets the user's location on every enter to the location page - but it doesnt save this until the user clicks on ```save```

Reply:
Go ahead with the change on PUT.

We can do a similar thing on the sign up screen. Grab GPS location upon visiting and converting it to an address to show on the field on screen, then we allow the user to change the address if it is incorrect. When they press signup, we will save the lat lon along with the newly created user. Do you think this is possible?

Reply:
So do you mean we'll be displaying the user's address somewhere in the sign up screen as well?

Reply:
yes